src/naive_bayes.cpp

```diff
@@ -0,0 +1,18 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
+
+#include <RcppMLPACK.h>				// MLPACK, Rcpp and RcppArmadillo
+
+#include <mlpack/methods/naive_bayes/naive_bayes_classifier.hpp> 	// particular algorithm used here
+
+// [[Rcpp::export]]
+Rcpp::List naive_bayes(const arma::mat& train, const arma::mat& test, const arma::Row<size_t>& labels, const int& classes) {
+    
+    arma::Row<size_t> results;
+
+    // Initialize with the default arguments.
+    mlpack::naive_bayes::NaiveBayesClassifier<> nbc(train, labels, classes);
+    
+    nbc.Classify(test, results);
+
+    return Rcpp::List::create(Rcpp::Named("results") = results);
```

Comment:
I just looked at the header. It does indeed have `Row<size_t>`.  I also saw lots of `rowvec` in MLPACK.

You need to see what works and what doesn't.   `RcppArmadillo` only knows `rowvec` and I was under the impression that this was aliases to `Row<size_t>`.  Maybe not.

I would prefer to have the _R interface_ use `rowvec` only.